Formula/mesa.rb

```diff
@@ -3,16 +3,11 @@ class Mesa < Formula
   homepage "https://dri.freedesktop.org"
   url "https://mesa.freedesktop.org/archive/mesa-17.2.3.tar.xz"
   sha256 "a0b0ec8f7b24dd044d7ab30a8c7e6d3767521e245f88d4ed5dd93315dc56f837"
-  revision 2
-
-  bottle do
-    root_url "https://linuxbrew.bintray.com/bottles-xorg"
-    rebuild 1
-    sha256 "b4c2ae6373c90eb23f5e8ed33d9757268802b793d56b17d6d58dd44ad9a48e32" => :x86_64_linux
-  end
```

Comment:
```
linuxbrew/xorg/mesa:
  * `bottle` is not defined
```
https://circleci.com/gh/Linuxbrew/homebrew-xorg/1091

Don't remove the bottle block.